lib/xlint.rb

```diff
@@ -28,6 +29,8 @@ def check_env
 
     def build_draft
       @comments = []
+      # GitDiffParser::Patches.parse(cp932 text) raises ArgumentError: invalid byte sequence in UTF-8
```

Comment:
I ran into the error when the git diff contained a PDF file. It's probable image files would also trigger the error.